tools/system_message.toit

```diff
@@ -49,6 +50,10 @@ main args:
             --short_help="The snapshot file of the program that produced the message",
         cli.OptionString "message" --short_name="m" --required
             --short_help="The base64-encoded message from the device",
+        cli.Flag "force_pretty"
+            --short_help="Force the report to use terminal graphics",
+        cli.Flag "force_plain"
```

Comment:
Nit: I strongly prefer flags to use dashes not underscores.

Reply:
I fixed it, and in the corresponding Jaguar change.

We should change cli so that either is accepted.